test/gfx/rgbgfx_test.cpp

```diff
@@ -281,6 +281,19 @@ class Png {
 };
 
 static char *execProg(char const *name, char * const *argv) {
+	auto formatArgv = [&argv] {
+		static std::string buf; // It's fine to use a static string:
```

Comment:
It may be "fine" (harmless), but is it for a particular reason? Is it just to avoid allocating a string every time `execProg` is called? If so, IMO just don't make it `static`, and then no need to `clear` either.

Reply:
No, it's so that the returned `c_str()` lives long enough. I'm not sure whether returning a `std::string` and then using *that*'s `.c_str()` is safe, or a UAF; so I chose something which I knew should work.

If making this return the `std::string` directly would work correctly, then by all means it should be done.

Reply:
Okay, thanks; then a comment to that effect would be helpful.

Reply:
(Ooh, just noticed that the no-arg anonymous function can be written as `[] {...}`, with no `()`. Neat!)